Sources/NIOHTTPClient/HTTPHandler.swift

```diff
@@ -272,8 +276,16 @@ public extension HTTPClient {
             }
         }
 
+        public func succeed(_ value: Response) {
```

Comment:
Does this need to be `public`? Should a user of the library be able to succeed/fail the task themselves?

Reply:
good point, this shouldn't be public, thanks!